l1-contracts/src.ts/deploy.ts

```diff
@@ -442,21 +443,52 @@ export class Deployer {
     }
   }
 
+  public async executeDirectOrGovernance(
+    useGovernance: boolean,
+    contract: Contract,
+    fname: string,
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    fargs: any[],
+    value: BigNumberish,
+    overrides?: Overrides,
+    printFileName?: string
```

Comment:
I know you copied this from me, so sorry, but could you rename it to printOperation and make it a boolean?